hack/ci/ci-setup-kubermatic-in-kind.sh

```diff
@@ -63,13 +63,11 @@ dockerd > /tmp/docker.log 2>&1 &
 echodate "Started Docker successfully"
 
 function docker_logs {
-  originalRC=$?
-  if [[ $originalRC -ne 0 ]]; then
+  if [[ $? -ne 0 ]]; then
     echodate "Printing Docker logs"
     cat /tmp/docker.log
     echodate "Done printing Docker logs"
   fi
-  return $originalRC
```

Comment:
Returning non-zero codes from traps actually prevents other traps from working. Also it's not necessary to return the original RC, the script's RC will stay the same even if all traps return 0.

This is why I removed the "get the original RC, then return it" logic in all our traps.